src/main/java/com/example/gamelist/GameMapper.java

```diff
@@ -1,13 +1,22 @@
 package com.example.gamelist;
 
+import com.example.gamelist.Game;
 import org.apache.ibatis.annotations.Mapper;
 import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
+
 
-import java.util.List;
 
 @Mapper
 public interface GameMapper {
 
+    //データを一括で取得する。　GET
     @Select("SELECT * FROM gamelists")
     List<Game> findAll();
+
+    @Select("SELECT * FROM gamelists WHERE id = #{id}")
+    Optional<Game> findById(int id);
+
+    @Update("UPDATE gamelist SET name = {#name} WHERE id = #{id}")
+    void updateGamelist(Gamelist gamelist);
```

Comment:
Gamelistがなぜか消えていますね。
Gameにしましょう。
```suggestion
    void updateGamelist(Game game);
```

Reply:
<img width="600" alt="スクリーンショット 2023-12-25 21 53 22" src="https://github.com/otyacazyu/gamelist/assets/143002834/4333dc2c-e589-4fa0-aab8-b71f8a64a7f2">
562a001aa33d8f165ffed70b50c25d5e9ffdfa79

gamelistをgameに変更しております。また、消えていたupdateGamelistを光らせました

Reply:
光らせたってなんですか？
あまり私使ったことないので意味が気になります。

Reply:
失礼しました。
updateGamelistの文字がグレーになっていたので、色が付いたことを光ったと表現していました。
他の表現の言葉が出てこなかった為、申し訳ありません

Reply:
なるほど。承知しました。

Reply:
こちらResolveでよいです。